processing/src/main/java/io/druid/query/aggregation/FilteredAggregatorFactory.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class FilteredAggregatorFactory implements AggregatorFactory
+public class FilteredAggregatorFactory extends AggregatorFactory
```

Comment:
This needs to override getMergingFactory as we could be merging different filters & different aggregators